src/lib/Scenes/Home/Components/Sales/index.tsx

```diff
@@ -12,43 +12,26 @@ import {
 } from "react-relay"
 import { RelayPaginationProp } from "react-relay"
 import styled from "styled-components/native"
-import { LotsByFollowedArtists } from "./Components/LotsByFollowedArtists"
+import LotsByFollowedArtists from "./Components/LotsByFollowedArtists"
 import { SaleList } from "./Components/SaleList"
 import { SectionHeader } from "./Components/SectionHeader"
 
-interface Props {
-  relay?: RelayPaginationProp
-  sales: Array<{
-    live_start_at: string | null
-  }>
-  viewer?: {
-    sale_artworks: {
-      pageInfo: object | null
-      edges: Array<{
-        node: object | null
-      }> | null
-    }
+class Sales extends React.Component<Props> {
+  static defaultProps = {
+    viewer: {
+      sales: [],
+    },
```

Comment:
What is this for? This shouldn’t be necessary.

Reply:
I'm probably being to overly defensive against crashes, but my thinking is if a prop is greater than one dot deep then it should be guarded and react makes guards very easy with defaults. But thinking about how MP works no matter what 'sales' will be be there. Will remove

Reply:
> But thinking about how MP works no matter what 'sales' will be be there. 

Exactly. And in these cases being overly defensive can actually mask legitimate bugs.